apps/greenhouse-management/src/lib/store.js

```diff
@@ -34,6 +36,7 @@ export default (options) => {
       isUrlStateSetup: false,
       assetsUrl: options.assetsUrl,
       apiEndpoint: options.apiEndpoint,
+      environment: options.environment,
```

Comment:
Do you also use the environment in the admin UIs?

Reply:
removed.